### PR TITLE
Bugfix/314 in user status response 401 unauthorized need 403 forbidden resolve

### PR DESCRIPTION
**Pull request by NazarVavrushchak**

### CODE REVIEW

**Review by vulook — APPROVED** [review]
That works

**Review by ZenukD — APPROVED** [review]

File: core/src/main/java/greencity/controller/UserController.java [hunk 1]
```diff
@@ -59,20 +64,26 @@ public class UserController {
...
     @PatchMapping("status")
     public ResponseEntity<UserStatusDto> updateStatus(
-        @Valid @RequestBody UserStatusDto userStatusDto, @ApiIgnore Principal principal) {
-        return ResponseEntity.status(HttpStatus.OK)
-            .body(
-                userService.updateStatus(
-                    userStatusDto.getId(), userStatusDto.getUserStatus(), principal.getName()));
+            @Valid @RequestBody UserStatusDto userStatusDto, @ApiIgnore Principal principal) {
+        if (principal == null) {
+            return ResponseEntity.status(HttpStatus.UNAUTHORIZED).build();
```
Comment (VitaliiChernenko) [inline core/src/main/java/greencity/controller/UserController.java]: In my opinion, this block if(principal == null) doesn't make any sense. In SecuritiConfig configured that only ADMIN can reach this endpoint. It means the principal can't be null!

Also, task 314 says "Sign-in us user". According to SecurityConfig this endpoint can't reach anybody except ADMIN.

I was trying to sign in as USER and enter valid data. And I received 401. But expected 403.
Please fix it.

**Review by VitaliiChernenko — CHANGES_REQUESTED** [review]

**Comment by VitaliiChernenko** [comment]
> That works

No, it doesn't)))

**Review by VitaliiChernenko — APPROVED** [review]

**Comment by VitaliiChernenko** [comment]
Now there is nothing to commit. Except for changing the formatting code, I do not see nothing in this PR. I guess the bug from task 314 was accidentally solved by another PR.  You can just close the PR without merging.

**Review by vulook — COMMENTED** [review]
It looks okay

**Pull request closed without merging**